### PR TITLE
Use different helm provider; Update Kubernetes version

### PR DESCRIPTION
**Pull request by andrewrynhard**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,42 +1,37 @@
...
 
-RUN curl https://bootstrap.pypa.io/get-pip.py > get-pip.py \
-    && python ./get-pip.py \
-    && pip install --upgrade \
-        cryptography  \
-        cffi \
-        pip \
-    && pip install \
-      ansible==${ANSIBLE_VERSION}
+RUN curl -LO https://download.opensuse.org/repositories/shells:fish:release:2/Debian_8.0/Release.key \
```
Comment (andrewrynhard) [inline Dockerfile]: Another shell to accommodate people that might use this shell.

**Comment by woz5999** [comment]
merge conflicts

**Comment by andrewrynhard** [comment]
That is strange.

**Comment by andrewrynhard** [comment]
@woz5999 fixed

**Pull request merged**